lib/taskana-core/src/main/java/pro/taskana/user/internal/UserMapper.java

```diff
@@ -60,6 +67,13 @@ public interface UserMapper {
   @InsertProvider(type = UserMapperSqlProvider.class, method = "insertGroups")
   void insertGroups(User user);
 
+  @InsertProvider(
+          type = UserMapperSqlProvider.class,
+          method = "insertPermissionsOracle",
+          databaseId = "oracle")
+  @InsertProvider(type = UserMapperSqlProvider.class, method = "insertPermissions")
+  void insertPermissions(User user);
```

Comment:
Is there an "insertPermissions" method for other databases besides Oracle? I couldn't find it

Reply:
yes, it is in the [UserMapperSqlProvider](https://github.com/nyuuyn/taskana/blob/f885c82fb3d8a59971b34ee0719edace2e1b06e9/lib/taskana-core/src/main/java/pro/taskana/user/internal/UserMapperSqlProvider.java#L75)